tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -133,7 +133,7 @@ class VerifiableConsumer(BackgroundThreadService):
         }
 
     def __init__(self, context, num_nodes, kafka, topic, group_id,
-                 max_messages=-1, session_timeout=30000, enable_autocommit=False,
+                 max_messages=-1, session_timeout_sec=30, enable_autocommit=False,
```

Comment:
I don't think this is a problem yet, but we should start thinking about these `Service` classes as at least semi-public interfaces. I know we rely on them in muckrake (although perhaps not this particular one yet) and I know others are starting to/planning to/want to be able to build tests on top of the pieces included in Kafka. While these definitely aren't the same as our client APIs, I think we should make an effort to provide some degree of compatibility.